runners/direct-java/src/main/java/org/apache/beam/runners/direct/TransformEvaluatorRegistry.java

```diff
@@ -45,43 +48,71 @@
  */
 class TransformEvaluatorRegistry implements TransformEvaluatorFactory {
   private static final Logger LOG = LoggerFactory.getLogger(TransformEvaluatorRegistry.class);
+
+  private static final String SPLITTABLE_PROCESS_URN =
```

Comment:
The `ProcessElements` transform is in runners-core (not construction). Certainly `ProcessFn` has to stay there, since it isn't a real `Fn` but just worker support code, but probably transforms should be in construction-core.

Anyhow `SplittableParDo` is where I have now put it, which is sort of close.